python/tk_multi_publish2/dialog.py

```diff
@@ -333,6 +346,14 @@ def _create_item_details(self, tree_item):
 
         self.ui.item_description_label.setText("Description")
         self.ui.item_comments.setPlainText(item.description)
+        
+        # if summary thumbnail is defined, item thumbnail should inherit it
+        # unless item thumbnail was set after summary thumbnail
+        if self._summary_thumbnail != None and not item._thumbnail_overrides_summary:
+           item.thumbnail = self._summary_thumbnail
+           # activate snaphot on inherited thumbnail, that way we can override it for a specific item
```

Comment:
Same here. I don't think this code should be enabling/disabling thumbnail manual creation.

Reply:
And similar comment about accessing `_thumbnail_overrides_summary`.